16 Lazy Loading/webpack.config.js

```diff
@@ -14,16 +14,32 @@ module.exports = {
       extensions: ['', '.js', '.ts', '.tsx']
   },
 
-  entry: [
-    './index.tsx',
-    './css/site.css',
-    '../node_modules/toastr/build/toastr.css',
-    '../node_modules/bootstrap/dist/css/bootstrap.css'
-  ],
+  entry: {
+    app: './index.tsx',
+    styles: [
+      './css/site.css',
+      '../node_modules/toastr/build/toastr.css',
+      '../node_modules/bootstrap/dist/css/bootstrap.css'
```

Comment:
We can work in two modes, having a global css file (we just provide the entry points and webpack generates the css chunk), or we can use imports. On this samples we have been using this approach.